mmv1/third_party/terraform/tests/data_source_dns_key_test.go

```diff
@@ -11,21 +11,41 @@ import (
 func TestAccDataSourceDNSKeys_basic(t *testing.T) {
 	t.Parallel()
 
-	dnsZoneName := fmt.Sprintf("data-dnskey-test-%s", RandString(t, 10))
+	dnsZoneName := fmt.Sprintf("tf-test-dnskey-test-%s", RandString(t, 10))
+
+	var kskDigest1, kskDigest2, zskPubKey1, zskPubKey2, kskAlg1, kskAlg2 string
 
 	VcrTest(t, resource.TestCase{
-		PreCheck:                 func() { testAccPreCheck(t) },
-		ProtoV5ProviderFactories: ProtoV5ProviderFactories(t),
-		CheckDestroy:             testAccCheckDNSManagedZoneDestroyProducer(t),
+		PreCheck:     func() { testAccPreCheck(t) },
+		CheckDestroy: testAccCheckDNSManagedZoneDestroyProducerFramework(t),
 		Steps: []resource.TestStep{
 			{
-				Config: testAccDataSourceDNSKeysConfig(dnsZoneName, "on"),
+				ExternalProviders: ProviderVersion450(),
```

Comment:
Good question! This was suggested by the DevEx team that we include them here. I have a concern that this will get to be a pretty heavy load if we add them to every test along the way. I can run it by them to see what their thoughts are. I'll add it to our discussion with them today! Thanks!